mmv1/third_party/terraform/utils/provider.go.erb

```diff
@@ -370,7 +370,7 @@ end     # products.each do
 				"google_apigee_sharedflow":                     ResourceApigeeSharedFlow(),
 				"google_apigee_sharedflow_deployment":          ResourceApigeeSharedFlowDeployment(),
 				"google_apigee_flowhook":                       ResourceApigeeFlowhook(),
-				"google_apigee_env_keystore_alias_pkcs12":	    ResourceApigeeEnvKeystoreAliasPkcs12(),
+				"google_apigee_keystores_aliases_pkcs12":       ResourceApigeeEnvKeystoreAliasPkcs12(),
```

Comment:
```suggestion
				"google_apigee_keystores_aliases_pkcs12":       ResourceApigeeKeystoreAliasPkcs12(),
```

Reply:
A little different, actually! `ResourceApigeeKeystoresAliasesPkcs12`. Replaced throughout- this still worked because I didn't touch function names initially but now I'd adjusted them all.